.github/workflows/build_all.yml

```diff
@@ -893,6 +893,93 @@ jobs:
           SLACK_ICON_EMOJI: ":heavy_check_mark:"
           SLACK_COLOR: "#00FF00"
           SLACK_FOOTER: ' '
+  build-deploy-arm:
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+      MAGMA_VERSION: "1.8.0"
+      MAGMA_AGW_CONTAINER_REGISTRY: "agw-test.artifactory.magmacore.org"
+    steps:
+      - uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e # pin@v2
+      - name: Run apt
+        run: sudo apt-get update && sudo apt -y upgrade
+      - name: setup pyenv
+        uses: "gabrielfalcao/pyenv-action@5327db2939908b2ef8f62d284403d678c4b611d0" # pin@v8
+        with:
+          default: 3.8.10
+      - uses: actions/setup-python@7f80679172b057fc5e90d70d197929d454754a5a # pin@v2
+        with:
+          python-version: '3.8.10'
+      - name: Setup SSH Keys known_hosts and sssh_config
+        env:
+          SSH_AUTH_SOCK: /tmp/ssh_agent.sock
+        run: |
+          ssh-agent -a $SSH_AUTH_SOCK > /dev/null
+          ssh-add - <<< "${{ secrets.LF_AWS_KEYPAIR }}"
+          mkdir ~/.ssh
+          touch ~/.ssh/gha_keypair.pem
+          echo "${{ secrets.LF_AWS_KEYPAIR }}" > ~/.ssh/gha_keypair.pem
+          sudo chmod 400 ~/.ssh/gha_keypair.pem
+          touch ~/.ssh/config
+          chmod 600  ~/.ssh/config
+          echo "Host *" >> ~/.ssh/config
+          echo "    ServerAliveInterval 240" >> ~/.ssh/config
+      - name: Install Dependencies
+        run: |
+          pip install ansible awscli boto3
+          sudo apt-get update
+      - name: Get the right tagging
+        if: github.event_name == 'push'
+        run: |
+          echo TAG="${MAGMA_VERSION}" >> $GITHUB_ENV
+          if [ "$GITHUB_REF" = "refs/heads/master" ]; then
+            echo TAG="${GITHUB_SHA:0:8}" >> $GITHUB_ENV
+          fi
+      # - name: Ansible config setup
+      #   run: |
+      #     echo "[defaults]" >> ~/.ansible.cfg
+      #     echo "host_key_checking = False" >> ~/.ansible.cfg
+      #     echo "ssh_args = -o ServerAliveInterval=60" >> ~/.ansible.cfg
+      - name: Run the ansible script to build on graviton
```

Comment:
This will probably only work on pushes (due to unavailable secrets in PRs), so we should guard it with an `if: github.event_name == 'push'`. In the long run we should adapt the job to support building-but-not-pushing when run without the necessary secrets.